wordpress-mobile/WordPress-Android
# Remove AnyFileUploadFeatureConfig.kt

Fixes #14263 

This PR removes the AnyFileUpload feature config because it's now enabled for 100% of users.

To test:
- Go to media screen 
- Click on the toolbar menu
- Notice that you correctly see the "Choose from device" option, not "Choose image/video from device"

PR submission checklist:
